app/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesJob.scala

```diff
@@ -49,6 +51,7 @@ class PollingNewMessagesJob @Inject() (
       .flatMap { lastActivityMap =>
         lastActivityMap.toSeq.traverse { case (ern, lastActivity) =>
           if (shouldUpdateMessages(lastActivity, fastIntervalTime, fastCutoffTime, slowIntervalTime)) {
+            logger.warn(s"[PollingNewMessagesJob] - Updating messages for ern: $ern")
```

Comment:
This will be _extremely_ noisy. Currently in staging this would result in 1000s of logs per minute. I think this should be removed, I also don't think it should be warn